node-tests/fixtures/component/component-addon-nested.ts

```diff
@@ -0,0 +1,10 @@
+import Component from '@ember/component';
+// @ts-ignore: Ignore import of compiled template
+import layout from '../../templates/components/foo/x-foo';
+
+export default class FooXFoo extends Component.extend({
+  // anything which *must* be merged to prototype here
+}) {
+  layout = layout;
+  // normal class body definition here
+};
```

Comment:
This fixture is actually just incorrect, as it hasn't been updated for the changes you made here!

```suggestion
export default class FooXFoo extends Component {
  layout = layout;
};
```